digitalocean_plugin/tasks.py

```diff
@@ -161,7 +82,7 @@ def start_droplet(droplet):
         start_droplet(create())
     else:
         ctx.logger.info("Starting existing droplet. Droplet id = '{0}'.".format(droplet_id))
-        d = get_droplet(droplet_id)
+        d = get_droplet(droplet_id, **_)
```

Comment:
Are you saying you slim down the list of possible inputs from **_ to just the one we need?